src/iks/iks.py

```diff
@@ -49,7 +50,7 @@ def handle_args(args):
             print("You did not specify an IKS cluster ID.")
             sys.exit(1)
 
-    iks_info = UserInfo.get_iks_info()
+    # iks_info = UserInfo.get_iks_info()
```

Comment:
This is needed for creating the DNS records

Reply:
This `iks_info` variable is never used anywhere.

Reply:
The variable itself is not used but get_iks_info() retrieves the ingress subdomain, which is needed for the DNS records

Reply:
idk who made it a variable tho, ig that's not necessary